.github/workflows/e2e_scripts/clean.sh

```diff
@@ -67,35 +68,6 @@ for LOG in $(gcloud logging logs list --project $PROJECT_ID --format="value(NAME
     gcloud logging logs delete $LOG --project $PROJECT_ID --quiet
 done
 
-# clear dashboards
-DASHBOARD_LIST="first_run"
-while [ -n "$DASHBOARD_LIST" ]; do
-  DASHBOARD_LIST=$(gcloud monitoring dashboards list --format="value(name)")
-  if [ -n "$DASHBOARD_LIST" ]; then
-    for DASHBOARD in $DASHBOARD_LIST; do
-      gcloud monitoring dashboards delete $DASHBOARD --quiet
-    done
-  fi
-done
-
-# clear policies
-POLICY_LIST="first_run"
-while [ -n "$POLICY_LIST" ]; do
-  POLICY_LIST=$(gcloud alpha monitoring policies list --format="value(name)")
-  if [ -n "$POLICY_LIST" ]; then
-    for POLICY in $POLICY_LIST; do
-      gcloud alpha monitoring policies delete $POLICY --quiet
-    done
-  fi
-done
-
-# clear notification channels
-CHANNEL_LIST="first_run"
-while [ -n "$CHANNEL_LIST" ]; do
-  CHANNEL_LIST=$(gcloud alpha monitoring channels list --format="value(name)")
-  if [ -n "$CHANNEL_LIST" ]; then
-    for CHANNEL in $CHANNEL_LIST; do
-      gcloud alpha monitoring channels delete $CHANNEL --quiet
-    done
-  fi
-done
+# clear monitoring
+python3 -m pip install -r ${WORKDIR}/requirements.txt
+python3 ${WORKDIR}/cleanup_monitoring.py $PROJECT_ID
```

Comment:
It looks like you only need the project name in the script, not the id. You can prefix this with "projects/", and only store the one variable

Reply:
True - I make this modification!